pkg/ciContext/ciContext.go

```diff
@@ -1,3 +1,9 @@
+/*
```

Comment:
I would avoid comments that tied to implementation because if anything in the implementation changes the comment might not be updated and then it just becomes confusing and loses its' meaning.